app/views/exchange_rates/index.html.erb

```diff
@@ -12,14 +12,24 @@
     <p class="govuk-body-l">Check the official <%= @period_list.year %> HMRC foreign currency exchange monthly rates.</p>
     <p class="govuk-body">Data is available in online (HTML), CSV and XML formats.</p>
 
-    <dt class="gem-c-metadata__term">From:</dt>
-    <dd class="gem-c-metadata__definition">
-      <strong><a href="https://www.gov.uk/government/organisations/hm-revenue-customs">HM Revenue &amp; Customs</a></strong>
-    </dd>
-    <dt class="gem-c-metadata__term">Published:</dt>
-    <dd class="gem-c-metadata__definition">
-      <%= @period_list.publication_date %>
-    </dd>
+    <div>
```

Comment:
I don't think you need the div here, since it didn't work and you've conditionally removed the publication date.

Reply:
I agree/